ert/data/record/_transmitter.py

```diff
@@ -146,6 +203,40 @@ def __init__(self, name: str, storage_path: Path):
         self._concrete_key = f"{name}_{uuid.uuid4()}"
         self._storage_uri = self._storage_path / self._concrete_key
 
+    async def get_recordtree_transmitters(
+        self,
+        trans_records: Dict[str, str],
+        record_type: RecordType,
+        _path: Optional[str] = None,
```

Comment:
Underscore?

Reply:
yes, `path` might be a function / module, therefore using `_`. If it is safe, I can do without.

Reply:
For collisions like that, I think they should be handled when/if it becomes a problem, and a trailing underscore should be used, since a leading one is associated with being private.